server/pyproject.toml

```
@@ -1,40 +1,53 @@
-[tool.poetry]
+[build-system]
+requires = ["uv-build >=0.6"]
+build-backend = "uv_build"
+
+[project]
 name = "testflinger"
 description = "Testflinger Server"
 readme = "README.rst"
 version = "1.1.0"                  # specify your version here
 authors = []
-packages = [
-    { include = "src" },
+#packages = [
+#    { include = "src" },
+#]
+requires-python = ">=3.10"
+dependencies = [
+    "apiflask >=2.1.1",
+    "bcrypt >=4.2.0",
+    "flask >=3.0.3",
+    "flask-pymongo >=3.0.0",
+    "gunicorn >=23.0.0",
+    "gevent >=24.2.1",
+    "prometheus-client >=0.21.0",
+    "prometheus-flask-exporter >=0.23.1",
+    "pyjwt >=2.8.0",
+    "pymongo <4.9.0",
+    "pyyaml >=6.0.1",
+    "requests >=2.31.0",
+    "sentry-sdk >= 2.0.1",
+    "urllib3 >=2.2.1",
 ]
 
+[project.scripts]
+client_credentials_admin = "src.testflinger.tools.client_credentials_admin:main"
```

The `project.scripts` sections should only use modules that are included in our package. In our case, that is anything under `src/` (we just have `testflinger`).

```suggestion
client_credentials_admin = "testflinger.tools.client_credentials_admin:main"
```

---

Modified reference for script in pyproject.toml